### DISCUSSION TITLE
Error executing verb "zip" in create_base_text_units: 'text'

### DISCUSSION THREAD

**Discussion by Morgan-rees34** [microsoft/graphrag#572] [Q&A]
Hi,

I'm trying to use GrahRAG by using csv files and am getting this error and not sure how to fix it:  Error executing verb "zip" in create_base_text_units: 'text'.
Not sure if relevant but using the .env and all the keys are being read except for the azure deployment, which it changes to the default values. (Got them saved as GRAPHRAG_EMBEDDING_DEPLOYMENT_NAME and GRAPHRAG_LLM_DEPLOYMENT_NAME)

Any help would be greatly appreciated, thanks!

**Comment by jasonmaoverlord** [comment]
I have solved the problem. If you modify settings.yaml well. your csv file must have a column named "text".
